app/assets/stylesheets/home.scss

```diff
@@ -20,18 +20,31 @@ body.home.index {
     }
 
     .leaflet-popup {
-      position: relative;
       background: white;
       padding: 1em;
       z-index: -100 !important;
-      margin-top: -4.75em;
+      margin-top: -(rem(104/2));
+      margin-left: 2em;
+
+      &:before {
+        content: "";
+        position: absolute;
+        left: -0.5em;
+        top: 50%;
+        margin-top: -0.55em;
+        @include triangle(1em, white, left);
+      }
 
       .leaflet-popup-close-button { display: none; }
       .leaflet-popup-content { padding-top: 0 !important; }
-      h5 {
+      h5, p {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 4<br>Each selector in a comma sequence should be on its own line